src/scss/_slider-input.scss

```diff
@@ -1,51 +1,104 @@
-$slider-handle-size: 2.2rem;
+$thumb-size: 20px;
+$thumb-color: $white;
+$thumb-focus-shadow: 0 0 8px $blue;
+$track-height: 4px;
+$track-color: $blue;
 
-.slider-input {
-  display: flex;
-  align-items: center;
+@mixin thumb {
+  -webkit-appearance: none;
+  box-shadow: 1px 1px 1px $black2, 0px 0px 1px $black2;
+  border: 1px solid $white1;
+  height: $thumb-size;
+  width: $thumb-size;
+  border-radius: 50%;
+  background: $thumb-color;
+  cursor: pointer;
+  margin-top: -(($thumb-size - $track-height) / 2);
+  transition: transform 100ms;
+  &:hover {
+    transform: scale(1.1);
+  }
 }
 
-.slider-input__bar {
-  position: relative;
-  height: $slider-handle-size;
+@mixin track {
   width: 100%;
-  touch-action: none;
+  height: $track-height;
+  cursor: pointer;
+  box-shadow: 1px 1px 1px $black2, 0px 0px 1px $black2;
+  background: $blue;
+  border-radius: 1.2px;
+}
+
 
-  // these are class names from the rc-slider lib
-  // we nedd the extra specificity here in order to actually style things
-  .rc-slider-rail,
-  .rc-slider-track {
-    position: absolute;
+.slider-input {
+  display: flex;
+  align-items: center;
+
+  input[type=range] {
```

Comment:
This should be `.slider-input__input` since that element has a className on it.

You should also NOT nest this inside `.slider-input` since it will add unnecessary specificity.

For example

```scss
.slider-input {
  .slider-input__input {
    ...
  }
}
```

Will compile into `.slider-input .slider-input__input`, but if you don't nest the rules you'll just get `.slider-input__input`.

This is good because it will allow you to override these styles later much more easily and you don't get trapped in a cascading style hell.